.circleci/config.yml

```diff
@@ -56,9 +56,11 @@ aliases:
     run:
       name: bundle install
       command: |
+        sudo gem update --system
         gem install bundler -v $(cat Gemfile.lock | tail -1 | tr -d " ")
         bundle config set --local path .bundle
         bundle check || bundle install --jobs=4 --retry=3
+        bundle env
```

Comment:
for troubleshooting and reporting potential issues. Can make a different PR.

Reply:
I like this here! I think this makes sense 😊